youtube_base/youtubers/feeds.py

```diff
@@ -0,0 +1,49 @@
+import os
+import markdown
+from django.contrib.sites.models import Site
+from django.contrib.syndication.views import Feed
+from django.template.defaultfilters import truncatewords_html
+from django.urls import reverse_lazy
+
+from .models import Youtuber
+
+
+class LatestYoutubersFeed(Feed):
+    """A Feed class that generates an RSS feed for the latest Youtubers."""
+    title = 'Українська база ютуберів.'
+    desription = 'Новий ютубер у нашій базі.'
+
+    def items(self):
+        """Return the Youtuber objects that should be included in the feed."""
+        return Youtuber.objects.all()[:5]
+
+    def link(self, item):
+        """Return the URL for the feed itself."""
+        site = Site.objects.get_current()
+        site.domain = os.environ.get('SITE_DOMAIN')
+        site.name = os.environ.get('SITE_NAME')
+        site.save()
+
+        domain = Site.objects.get_current().domain
```

Comment:
27-28 and 33-34 lines are almost equal.

Let's detach a method

def getUrl(self, url_parameters):
    domain = Site.objects.get_current().domain
    item_link = 'http://' + domain + url_parameters